Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -210,7 +210,7 @@ JITCode* JITCode::dfg()
     return this;
 }
 
-void JITCode::shrinkToFit(const ConcurrentJSLocker&)
+void JITCode::shrinkToFit()
 {
```

Comment:
Do we know why this originally had a lock?

Reply:
I think this was originally just randomly used, but in fact this is not necessary.